queries/futures/useGetStatsVolumes.ts

```diff
@@ -30,14 +24,16 @@ export const useGetStatsVolumes = () => {
 
 	const query = async () => {
 		try {
-			const response = await getFuturesHourlyStats(
+			const response = await getFuturesAggregateStats(
 				futuresEndpoint,
 				{
 					first: 999999,
 					orderBy: 'timestamp',
 					orderDirection: 'asc',
 					where: {
+						period: `${PERIOD_IN_SECONDS.ONE_DAY}`,
 						timestamp_gt: minTimestamp,
+						asset: '0x',
```

Comment:
Just wondering what this 0x asset is here?

Reply:
When I create aggregates I create one for each asset, then create this `0x` one which is the total of all assets

Reply:
Ok yea I see, might be worth creating a constant to help document, like `const AGGREGATE_ASSET = '0x'`? Other than that all good